### PR TITLE
Mark RUSTSEC-2020-0146 as unsound

### PR DESCRIPTION
**Pull request by xfix**
I didn't realize this was an option because it wasn't in example advisory.

### CODE REVIEW

File: crates/generic-array/RUSTSEC-2020-0146.md [hunk 1]
```diff
@@ -3,9 +3,8 @@
 id = "RUSTSEC-2020-0146"
 package = "generic-array"
 date = "2020-04-09"
+informational = "unsound"
 url = "https://github.com/fizyk20/generic-array/issues/98"
-categories = ["memory-corruption"]
```
Comment (tarcieri) [inline crates/generic-array/RUSTSEC-2020-0146.md:7]: I'd suggest leaving this one

**Comment by Shnatsel** [comment]
`informational = "unsound"` downgrades this from an error to a warning. Is that intentional? Since this allows use-after-free, I'd suggest leaving it as a hard error.

**Comment by tarcieri** [comment]
@Shnatsel hmm, maybe you're right. Seems this is a bigger problem than I thought.

**Comment by Shnatsel** [comment]
@xfix could you elaborate on the reasoning for marking it informational? (We do need to document that field; that's #751)

**Comment by Shnatsel** [comment]
I meant to open a pull request reverting this for discussion and instead accidentally committed to master :facepalm:

**Comment by xfix** [comment]
I had no idea about policies, if it shouldn't be informational, that's fine with me.

**Comment by Shnatsel** [comment]
Yeah, we really should document that better. Thanks!

**Pull request merged**